demos/src/count.js

```diff
@@ -0,0 +1,7 @@
+const oComments = require('../../main');
+
+document.addEventListener('DOMContentLoaded', function() {
+	document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
+});
+
+window.oComments = oComments;
```

Comment:
I have no idea. To be completely honest, I don't know what this line of code does. It was something I copy/pasted from o-comments. Should I delete it?

Reply:
Yeah it can be deleted 👍 [window](https://developer.mozilla.org/en-US/docs/Web/API/Window) is a global object, by setting `window.oComments = oComments` any code could access `oComments` by calling `window.oComments`, which we don't need for the demo to work

Reply:
Nice! I love deleting things.